server/schedule/operator/step.go

```diff
@@ -406,7 +406,7 @@ type PromoteLearner struct {
 // ConfVerChanged returns the delta value for version increased by this step.
 func (pl PromoteLearner) ConfVerChanged(region *core.RegionInfo) uint64 {
 	peer := region.GetStoreVoter(pl.ToStore)
-	return typeutil.BoolToUint64(peer.GetId() == pl.PeerID)
+	return typeutil.BoolToUint64(peer.GetId() == pl.PeerID && peer.GetRole() == metapb.PeerRole_Voter)
```

Comment:
Currently, IncomingVoter and DemotingVoter are also considered Voter, so additional checks are required.

Reply:
maybe we should add these comments to code, which is convenient to read code.

Reply:
add some comment